x-pack/legacy/plugins/code/public/components/integrations/integrations.scss

```diff
@@ -0,0 +1,43 @@
+.integrations__container {
+  padding: $euiSize;
+}
+
+.integrations__frame {
+  margin: $euiSizeS 0;
+}
+
+.integrations__code {
+  @include euiCodeFont;
+}
+
+.integrations__link--external {
+  margin-left: $euiSizeS;
+}
+
+.integrations__preposition {
+  margin: 0 $euiSizeS;
+  color: $euiColorMediumShade;
+}
+
+.integrations__button-icon {
+  padding: $euiSizeXS;
+  background-color: $euiColorLightestShade;
+  border: 1px solid $euiColorLightShade;
+}
+
+.integrations__snippet-info {
+  margin-bottom: $euiSizeS;
+}
+
+.integrations__snippet-title {
+  margin-bottom: $euiSizeS;
+}
+
+.integrations__text--bold {
+  font-weight: $euiFontWeightBold;
+}
+
+.integrations__popover {
+  margin-bottom: 1rem;
+  width: 300px;
+}
```

Comment:
Same comment from https://github.com/elastic/kibana/pull/47180 - Probably want to scope this to `codeIntegrations__`

Reply:
Yep, #47180 is getting merged immediately after this one, and it's based on this, so the rename's gonna happen there.